src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/components/ArrayBuilderCards.jsx

```diff
@@ -0,0 +1,295 @@
+/* eslint-disable @department-of-veterans-affairs/prefer-button-component */
```

Comment:
File renamed from previous `ArrayBuilderSummaryCardList.jsx`, and did a bunch of refactor here